### PR TITLE
Add Anytype EVM network to networks.json

### PR DESCRIPTION
**Pull request by AnthonyAkentiev**
Changes proposed in this pull request:
- Add support for Anytype EVM network

Thank you very much! You are building an awesome product!

### CODE REVIEW

**Comment by AnthonyAkentiev** [comment]
@ChaituVR Hello. Could you please check my PR? Can't set you as a reviewer...

File: src/networks.json [hunk 1]
```diff
@@ -1624,5 +1624,19 @@
...
+  },
+  "1701": {
+    "key": "1701",
+    "name": "Anytype EVM Chain",
+    "shortName": "AnytypeChain",
+    "chainId": 1701,
+    "network": "mainnet",
+    "multicall": "0x4542a07dE7f8A1cEB85aEcBFD0d5338e9886EF49",
+    "rpc": [
+      "https://geth.anytype.io"
```
Comment (ChaituVR) [inline src/networks.json:1636]: HI @AnthonyAkentiev I am getting CORS issue with this node, this node won't work on the front end.?

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/networks.json (hunk 1 above)
Comment (AnthonyAkentiev) [reply src/networks.json:1636]: @ChaituVR Please check again. I've set --http.corsdomain '*' for now. 
What domain should I ad? Is it**snapshot.org**?

Comment (ChaituVR) [reply src/networks.json:1636]: I tried again but it still doesn't work, would be great if you can add `snapshot.org` and `snapshot-networks.on.fleek.co` . is there any restriction based on the IP/rate limit to this node?

Comment (AnthonyAkentiev) [reply src/networks.json:1636]: @ChaituVR fixed. please try again
sorry

**Comment by AnthonyAkentiev** [comment]
@ChaituVR I've fixed everything. Please try again. Thank you.

File: src/networks.json (hunk 1 above)
Comment (ChaituVR) [reply src/networks.json:1636]: @AnthonyAkentiev I still face the same issue. it works fine 

```JSON
"1701": {
    "key": "1701",
    "name": "Anytype EVM Chain",
    "shortName": "AnytypeChain",
    "chainId": 1701,
    "network": "mainnet",
    "multicall": "0x4542a07dE7f8A1cEB85aEcBFD0d5338e9886EF49",
    "rpc": [
      "https://geth.anytype.io"
    ],
    "explorer": "https://explorer.anytype.io",
    "start": 1126886,
    "logo": "ipfs://QmaARJiAQUn4Z6wG8GLEry3kTeBB3k6RfHzSZU9SPhBgcG"
  },
  ```

Comment (ChaituVR) [reply src/networks.json:1636]: @AnthonyAkentiev I still face the same issue. it works fine when i use postman, but fails in chrome browser

Comment (AnthonyAkentiev) [reply src/networks.json:1636]: @ChaituVR 
Weird!

```
curl https://geth.anytype.io/ -H "Host: [geth.anytype.io](http://geth.anytype.io/)" -X POST -H "Content-type: application/json" -v

Response:
< HTTP/1.1 200 OK
< Date: Mon, 24 Oct 2022 16:52:59 GMT
< Content-Type: application/json
< Content-Length: 0
< Connection: keep-alive
< Server: nginx/1.18.0 (Ubuntu)
< Access-Control-Allow-Origin: *
< X-Host: ip-172-31-35-84.eu-central-1.compute.internal
```

Could you please post which request fails?
You can copy it directly from the Chrome's developers console ("Networking" tab). Thank you.

Comment (AnthonyAkentiev) [reply src/networks.json:1636]: @ChaituVR thx for you help.
I've fixed everything, please try again.

here is the debug page with a button that sends the POST request -> https://evm.anytype.io/debug.html
you can click it to try

worked in the Chrome too

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**